test/java-bdk.spec.js

```diff
@@ -55,9 +55,11 @@ it('Generate 2.0 spring boot gradle', () => {
                 'build.gradle',
                 path.join(BASE_JAVA, PACKAGE_DIR, 'BotApplication.java'),
                 path.join(BASE_JAVA, PACKAGE_DIR, 'GifFormActivity.java'),
-                path.join(BASE_JAVA, PACKAGE_DIR, 'RealTimeEventsComponent.java'),
-                path.join(BASE_RESOURCE, 'rsa/privatekey.pem'),
-                path.join(BASE_RESOURCE, 'rsa/publickey.pem'),
+                path.join(BASE_JAVA, PACKAGE_DIR, 'OnUserJoinedRoomListener.java'),
+                path.join(BASE_JAVA, PACKAGE_DIR, 'GifSlashHandler.java'),
+                // path.join(BASE_RESOURCE, 'rsa/privatekey.pem'),
```

Comment:
Should we remove that?

Reply:
I'll fix that later, I don't know why it is not working...